disnake/gateway.py

```diff
@@ -639,7 +639,19 @@ async def received_message(self, raw_msg: Union[str, bytes], /) -> None:
         except KeyError:
             _log.debug("Unknown event %s.", event)
         else:
-            func(data)
+            try:
+                func(data)
+            except BaseException as e:
```

Comment:
should this just use Exception?

Reply:
yup, not sure why I used `BaseException`, there's no point in using it here
d79387050dca5bca63225bda8343cdc9481baf49